test/e2e/minikube.test.ts

```diff
@@ -24,11 +29,44 @@ describe('e2e test', () => {
           .and.to.contain('Command server:start has completed successfully')
       })
     test
-      .skip()
-      .stdout()
+      .it('Obtain access_token from keycloak.', async () => {
+        const token = await helper.Access_Token(execution_platform)
+        process.env.CHE_ACCESS_TOKEN = token
+      })
+  })
+})
+
+describe('Workspace creation, list, start, inject, delete. Support stop and delete commands for Eclipse Che server', () => {
+  const wait = (ms = 10) => new Promise(resolve => setTimeout(resolve, ms))
+
+  // !TODO ADD coverage for next commands: workspace:delete and workspace:inject
+  describe('Create Workspace', () => {
+    test
+      .stdout({ print: true })
+      .command(['workspace:create', '--start', '--devfile=test/e2e/util/devfile-example.yaml'])
+      .exit(0)
+      .it('Create a workspace', async () => {
+        await wait(160000)
+        const workspace = await helper.WorkspaceID(execution_platform)
+        expect(workspace[0].status).to.contain('RUNNING')
```

Comment:
If workspace list is empty test will fail with unclear error. Let's check size first.